aws_lambda_powertools/logging/logger.py

```diff
@@ -292,6 +273,29 @@ def structure_logs(self, append: bool = False, **kwargs):
                 handler.setFormatter(JsonFormatter(**self._default_log_keys, **kwargs))
 
 
+def _get_log_level(level: Union[str, int]) -> Union[str, int]:
+    """ Returns preferred log level set by the customer in upper case """
+    if isinstance(level, int):
+        return level
+
+    log_level: str = level or os.getenv("LOG_LEVEL")
+    log_level = log_level.upper() if log_level is not None else logging.INFO
+
+    return log_level
+
+
+def _get_caller_filename():
```

Comment:
make this a classmethod, not a separate function - it's still part of the encapsulated logger class logic

Reply:
@heitorlessa should I use `@staticmethod` then ?

Reply:
sorry, yes, that's what I meant - `staticmethod`.